compose.yaml

```diff
@@ -1,16 +1,24 @@
 services:
   postgres:
     image: 'postgres:latest'
+    volumes:
+      - mydatabase_data:/var/lib/postgresql/data/
     environment:
-      - 'POSTGRES_DB=mydatabase'
-      - 'POSTGRES_PASSWORD=secret'
-      - 'POSTGRES_USER=myuser'
+      - 'POSTGRES_DB=postgres'
+      - 'POSTGRES_PASSWORD=orderhere'
+      - 'POSTGRES_USER=postgres'
+
+    # To avoid clashes with pgAdmin default server
     ports:
-      - '5432'
+      - '5570:5432'
+
   rabbitmq:
     image: 'rabbitmq:latest'
     environment:
-      - 'RABBITMQ_DEFAULT_PASS=secret'
-      - 'RABBITMQ_DEFAULT_USER=myuser'
+      - 'RABBITMQ_DEFAULT_PASS=orderhere'
+      - 'RABBITMQ_DEFAULT_USER=rabbitmq'
     ports:
-      - '5672'
+      - '5672:5672'
```

Comment:
Very good, but we have not used any feature of rabbitMQ yet, which will cause additional memory usage. Please comment out this part for now